rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stat_cfg-v0.0.1.json

```diff
@@ -0,0 +1,175 @@
+  "required": [
+    "$schema",
+    "pid",
+    "indicator",
+    "owner",
+    "organisation",
+    "category",
+    "frequency",
+    "description",
+    "name",
+    "status"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate stats cfg records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://bib.rero.ch/schemas/stats_cfg/stat_cfg-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Stat ID.",
+      "type": "string",
+      "minLength": 1
+    },
+    "indicator": {
+      "title": "Statistics indicator",
+      "type": "string",
+      "enum": [
+        "number_of_documents",
+        "number_of_serial_holdings",
+        "number_of_items",
+        "number_of_patrons",
+        "number_of_active_patrons",
+        "number_of_ill_requests",
+        "number_of_deleted_items",
+        "number_of_checkins",
+        "number_of_checkouts",
+        "number_of_renewals",
+        "number_of_requests"
+      ]
+    },
+    "dist1": {
+      "title": "Statistics distribution 1",
+      "type": "string",
+      "enum": [
+        "time_range_month",
+        "time_range_year",
+        "library",
+        "item_owning_library",
+        "imported",
+        "location",
+        "type",
+        "gender",
+        "role",
+        "patron_postal_code",
+        "status",
+        "item_location",
+        "patron_type",
+        "document_type",
+        "transaction_channel"
+      ]
+    },
+    "dist2": {
+      "title": "Statistics distribution 2",
+      "type": "string",
+      "enum": [
+        "time_range_month",
+        "time_range_year",
+        "library",
+        "item_owning_library",
+        "imported",
+        "location",
+        "type",
+        "gender",
+        "role",
+        "patron_postal_code",
+        "status",
+        "item_location",
+        "patron_type",
+        "document_type",
+        "transaction_channel"
+      ]
+    },
+    "owner": {
+      "title": "System librarian pid, the creator of the configuration.",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "$ref"
+      ],
+      "properties": {
+        "$ref": {
+          "title": "Patron URI",
+          "type": "string",
+          "pattern": "^https://bib.rero.ch/api/patrons/.*?$"
+        }
+      }
+    },
+    "organisation": {
+      "title": "Organisation",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "$ref"
+      ],
+      "properties": {
+        "$ref": {
+          "title": "Organisation URI",
+          "type": "string",
+          "pattern": "^https://bib.rero.ch/api/organisations/.*?$"
+        }
+      }
+    },
+    "category": {
+      "title": "Category of the statistics report configuration.",
+      "type": "string",
+      "enum": [
+        "Administration",
+        "Catalogue",
+        "Circulation",
+        "User management"
```

Comment:
values should be lowercase and snakecase, `User management` --> `user_management`